File: DSA/DakshSethi_590012025/Day_1_Question_3.c
#include <stdio.h>
 
int main()
{
    int size;
    printf("enter size of the array \n");
    scanf("%d",&size);

    int arr[size];
    printf("enter array elements\n");

    for(int i=0;i<size;i++)
    scanf("%d",&arr[i]);
    int sum=0;

    for(int i=0;i<size;i++)
    sum+=arr[i];
 
    printf("sum of elements of array is %d \n",sum);
    return 0;
}

File: DSA/AnshiAgrawal_500124498/Day_1/Day1Question3.java
import java.util.Scanner;

public class Day1Question3 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter size of array:");
        int size=sc.nextInt();
        int[] arr=new int[size];
        int i,sum=0;
        for( i=0; i<size;i++){
            System.out.println("Enter "+(i+1)+ " Element:");
            arr[i]=sc.nextInt();
            sum=sum+arr[i];
        }
        for (i=0;i<size;i++){
            System.out.print(arr[i]+(i<size-1?",":"\n"));
        }
        System.out.println("Sum of Elements of array:"+sum);
    }
}
